iml-wire-types/src/lib.rs

```diff
@@ -2079,3 +2081,217 @@ impl PartialEq for LdevEntry {
         self.label == other.label
     }
 }
+
+#[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum LndType {
+    Tcp,
+    O2ib,
+}
+
+#[derive(Debug, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum LinuxType {
+    Ethernet,
+    Ether,
```

Comment:
What is the difference between `Ethernet` and `Ether`?

Reply:
Both types were included in the original source and both are mapped to "tcp".

Reply:
Why do we need to have both of them?

Reply:
I believe it depends on the output. I don't think we are guaranteed that it will always be `ether` or always be `ethernet`.

Reply:
From my experience, however, i've only seen `ether`.

Reply:
Don't we always convert both to tcp in the output though?

Reply:
Yes, both are converted to "tcp" for the `network_interface` table.

Reply:
I can take the `Ethernet` variant out but I don't think it hurts to have it there to cover that case in the event that it is used in the output.

Reply:
I think it could be confusing to have two variants that mean the same thing.